gnovm/stdlibs/std/emit_event.go

```diff
@@ -19,14 +19,27 @@
 
 	_, pkgPath := currentRealm(m)
 	fnIdent := getPrevFunctionNameFromTarget(m, "Emit")
+	prevFunc := getPrevFunctionNameFromTarget(m, fnIdent) // get the previous function name of the function that called Emit
+
+	prevAddr, prevPath := X_getRealm(m, 1)
```

Comment:
Just curious, do we use `X_getRealm` calls a lot with `1`?
If so, we can consider making this a named constant

Reply:
Please no, let's instead add a helper `func prevRealm(...) Realm { return X_getRealm(m, 1) }`. We have this for `currentRealm` as well.

Reply:
a1d9c905fe99b84370d729d9b9dafa156db9332d